mkdd_editor.py

```diff
@@ -2787,6 +2790,27 @@ def action_update_position(self, event, pos):
 
         self.statusbar.showMessage(display_string)
 
+    def dragEnterEvent(self, event):
+        mime_data = event.mimeData()
+        if mime_data.hasUrls():
+            url = mime_data.urls()[0]
+            filepath = url.toLocalFile()
+            ext = os.path.splitext(filepath)[1].lower()
+            if ext in (".bol", ".arc", ".bmd", ".bco"):
+                event.acceptProposedAction()
+
+    def dropEvent(self, event):
+        mime_data = event.mimeData()
+        if mime_data.hasUrls():
+            url = mime_data.urls()[0]
+            filepath = url.toLocalFile()
+            ext = os.path.splitext(filepath)[1].lower()
+            if ext in (".bol", ".arc"):
+                self.button_load_level( False, filepath, update_config = False)
+            elif ext == ".bco":
+                self.load_optional_bco(filepath)
+            elif ext == ".bmd":
+                self.load_optional_bmd(filepath)
```

Comment:
Nit: Add two blank lines between `GenEditor` and `find_file()`.

PEP 8 recommends two blank lines between top-level module functions. In the [**Blank Lines**](https://peps.python.org/pep-0008/#blank-lines) section:

> Surround top-level function and class definitions with two blank lines.